src/libstd/sys/common/unwind/mod.rs

```diff
@@ -211,6 +215,10 @@ pub fn begin_unwind_fmt(msg: fmt::Arguments, file_line: &(&'static str, u32)) ->
 }
 
 /// This is the entry point of unwinding for panic!() and assert!().
+#[doc(hidden)]
+#[unstable(feature = "libstd_sys_internals",
+           reason = "this function is considered exported for some reason",
```

Comment:
I'll improve the messages/tweak stability for the items you've mentioned.
("for some reason" is a stub, the messages can always be improved if better reasons are available)